config/exchanges.json

```diff
@@ -1724,9 +1724,7 @@
       -115.846,
       35.962
     ],
-    "parsers": {
-      "exchange": "US_CA.fetch_exchange"
-    },
+    "parsers": {},
```

Comment:
This change for US->US-CA should probably not be here?

Reply:
Ok it should be fixed now, there's still a problem with US->US-CA not working though (json serialization error I think).